tooling/debugger/tests/debug.rs

```diff
@@ -144,6 +144,18 @@ mod tests {
             VecDeque::from(["foo(x);", "bar(y);", "fn bar(y: Field) {"]),
             VecDeque::from(["foo(x);", "bar(y);", "assert(y != 0);"]),
         ];
+
+        // Try to use relative paths if possible, otherwise the test breaks when run from the repository root
+        let at_filename = if let Ok(current_dir) = std::env::current_dir() {
+            if let Ok(stripped) = test_program_path.strip_prefix(current_dir) {
+                format!("At {}", stripped.display())
+            } else {
+                format!("At {}", test_program_dir)
+            }
+        } else {
+            format!("At {}", test_program_dir)
+        };
```

Comment:
```suggestion
        let at_filename = std::env::current_dir()
            .ok()
            .and_then(|dir| test_program_path.strip_prefix(&dir).ok())
            .map(|stripped| format!("At {}", stripped.display()))
            .unwrap_or_else(|| format!("At {}", test_program_dir));
```
nit: we can reduce nesting here with a clear single fallback